packages/api/lib/writeRecords/write-granules.js

```diff
@@ -262,21 +262,29 @@ const _writeGranuleFiles = async ({
       knex,
     });
   } catch (error) {
+    const failures = [];
     if (!isEmpty(workflowError)) {
       log.error(`Logging existing error encountered by granule ${granuleId} before overwrite`, workflowError);
+      failures.push(workflowError);
     }
     log.error('Failed writing files to PostgreSQL, updating granule with error', error);
     const errorObject = {
       Error: 'Failed writing files to PostgreSQL.',
       Cause: error.toString(),
     };
+    failures.push(errorObject);
+
+    const errorsObject = {
+      Errors: JSON.stringify(failures),
```

Comment:
```suggestion
      errors: JSON.stringify(errors),
```

Nit, so take it or leave it

Reply:
[`65786c0` (#2567)](https://github.com/nasa/cumulus/pull/2567/commits/65786c0d912d5deb00162f4b6982906b60c341d8#diff-705c5c617b45462fde4dd4c0b8b964f95e91ec857b774841daebe5144d9dd990R278)